src/main/java/com/couchbase/client/encryption/errors/CryptoException.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2020 Couchbase, Inc.
+ *
+ * Use of this software is subject to the Couchbase Inc. Enterprise Subscription License Agreement v7
+ * which may be found at https://www.couchbase.com/ESLA01162020.
+ */
+
+package com.couchbase.client.encryption.errors;
+
+public class CryptoException extends RuntimeException {
```

Comment:
I think it should extend the CouchbaseException since it can then have an error context in the future and we need to standardize them across sdks